weekly-scan/template.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: "weekly-scan-job-template"
+objects:
+  - apiVersion: "v1"
+    kind: "BuildConfig"
+    metadata:
+      name: ${PIPELINE_NAME}
+    spec:
+      strategy:
+        type: "JenkinsPipeline"
+        jenkinsPipelineStrategy:
+          jenkinsfile: |
+            properties([
+                pipelineTriggers([
+                    cron('0 23 * * 6'),
+                ])
+            ])
+            podTemplate(
+                cloud: 'openshift',
+                name: 'ccp-pipeline',
+                label: 'ccp-pipeline',
+                serviceAccount: 'jenkins',
+                containers: [
+                  containerTemplate(
+                    name: 'jnlp',
```

Comment:
TBH! for segregation purpose I'd not use the same container name as that of build containers, probably add `wscan` here as well.

Reply:
Fixed